src/tests/ftest/launch.py

```diff
@@ -1183,10 +1183,9 @@ def archive_config_files(avocado_logs_dir, args):
     print("Archiving config files from {} in {}".format(host_list, destination))
 
     # Copy any config files
-    base_dir = get_build_environment(args)["PREFIX"]
-    configs_dir = get_temporary_directory(args, base_dir)
+    configs_dir = os.path.join(os.sep, "etc", "daos")
```

Comment:
Hrm.  How does this operate in a _local-install_ environment? AFAIU we have not yet sun-setted that.  I suspect we want to keep it for developers running locally and possibly for code coverage test runs that need a "local-install"-like environment.

Reply:
I've updated the changes to gather config files from either location:

- https://github.com/daos-stack/daos/blob/9f7773bab43e82a09e4d9dae75dcea45172ddd6c/src/tests/ftest/launch.py#L967-L975
- https://github.com/daos-stack/daos/blob/9f7773bab43e82a09e4d9dae75dcea45172ddd6c/src/tests/ftest/launch.py#L977-L983